doc/python_reference.rst

```diff
@@ -305,13 +305,10 @@ Projections:
    :toctree: generated/
 
    Layout
-   Montage
    DigMontage
    fix_mag_coil_types
-   read_montage
    read_polhemus_fastscan
    get_builtin_montages
-   read_dig_montage
```

Comment:
We should backport these `python_reference.rst` removals to the `maint/0.19` branch

Reply:
f21ad8b0ea98a635c48cc53847cd94547eca67b8 should also be ported since it fixes part of the doc, that was still pointing to `:class:mne.channels.Montage`